### PR TITLE
Expose APIs and Events under System Instances in ORD Service

### PR DESCRIPTION
**Pull request by ivantenevvasilev**
**Description**

Changes proposed in this pull request:
- bump ORD service version
- add e2e test that validates systems instances have exposed APIs and Events

**Related issue(s)**
- kyma-incubator/ord-service#116

**Pull Request status**

- [x] [N/A] Implementation
- [x] [N/A] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [x] [N/A] Mocks are regenerated, using the automated script

### CODE REVIEW

File: tests/ord-service/tests/api_test.go [hunk 1]
```diff
@@ -357,6 +357,22 @@ func TestORDService(stdT *testing.T) {
 			require.Equal(t, testData.appInput.Name, gjson.Get(respBody, "value.0.title").String())
 			require.Equal(t, *testData.appInput.Description, gjson.Get(respBody, "value.0.description").String())
 		})
+		t.Run(fmt.Sprintf("Requesting System Instances with apis for tenant %s returns them as expected", testData.msg), func(t *testing.T) {
+			respBody := makeRequestWithHeaders(t, testData.client, testData.url+"/systemInstances?$format=json&$expand=apis", testData.headers)
```
Comment (nyordanoff) [inline tests/ord-service/tests/api_test.go:361]: It's more conventional to be:

`/systemInstances?$expand=apis&$format=json`

Same below for the events.

**Review by PetarTodorovv — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Pull request merged**